stacker/context.py

```diff
@@ -70,7 +71,11 @@ def load_config(self, conf_string):
             self.bucket_name = bucket_name
         tags = self.config.get("tags", None)
         if tags is not None:
-            self.tags = dict([(str(tag_key), str(tag_value)) for tag_key, tag_value in tags.items()])
+            self.tags = dict([(str(tag_key), str(tag_value)) for tag_key,
+                              tag_value in tags.items()])
+        lookups = self.config.get("lookups", {})
+        for key, handler in lookups.iteritems():
```

Comment:
Will all lookups need to be registered in the config, or will some be loaded automatically?  If they do have to be loaded, is there any reason why you chose that over loading all of them automatically?